src/Containers/RightSideBar/EditText.tsx

```diff
@@ -0,0 +1,253 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Select } from '@Inputs/Select';
+import { Bold } from '@styled-icons/boxicons-regular/Bold';
+import { Italic } from '@styled-icons/boxicons-regular/Italic';
+import { Underline } from '@styled-icons/boxicons-regular/Underline';
+import { TextColor } from '@styled-icons/foundation/TextColor';
+import { AlignMiddle } from '@styled-icons/boxicons-regular/AlignMiddle';
+import { AlignLeft } from '@styled-icons/boxicons-regular/AlignLeft';
+import { AlignRight } from '@styled-icons/boxicons-regular/AlignRight';
+
+// API KEY: AIzaSyBW6afsWt6KM6yvzXgfJ9UmUGImHvedZbc
+// STYLED COMPONENTS
+const Wrapper = styled.div``;
+const StyledSelect = styled(Select)`
```

Comment:
StyledSelect and FontStyledSelect are the same, so no need for unnecessary repeats